src/browser/bounds_changed_state_tracker.js

```diff
@@ -215,36 +276,25 @@ function BoundsChangedStateTracker(uuid, name, browserWindow) {
                     let hwndToId = {};
 
                     const { flag: { noZorder, noSize, noActivate } } = windowTransaction;
-                    const flags = noZorder + noSize + noActivate;
+                    let flags;
+                    if (changeType === 1) {
+                        // this may need to change to 1 or 2 if we fix functionality for changeType 2
+                        flags = noZorder + noActivate;
+                    } else {
+                        flags = noZorder + noSize + noActivate;
+                    }
 
                     WindowGroups.getGroup(groupUuid).filter((win) => {
                         win.browserWindow.bringToFront();
                         return win.name !== name;
                     }).forEach((win) => {
-                        let { x, y, width, height } = win.browserWindow.getBounds();
+                        let winBounds = win.browserWindow.getBounds();
+                        let { x, y, width, height } = (changeType === 1) ? handleGroupedResize(boundsCompare, delta, cachedBounds, win): winBounds;
```

Comment:
Both should be `const`s?

Reply:
winBounds yes other no - may be redefined below.  Requested Changes made.